### PR TITLE
chore: add manifest files for event persister dwh

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: manifests/bucketeer/values.yaml [hunk 1]
```diff
@@ -2513,3 +2513,133 @@ web-gateway:
...
+    pullPolicy: IfNotPresent
+  nameOverride: "event-persister-dwh"
+  fullnameOverride: "event-persister-evaluation-events-dwh"
+  namespace: default
+  env:
+    project:
+    logLevel: info
+    port: 9090
+    metricsPort: 9002
+    bigtableInstance:
```
Comment (masaaania) [inline manifests/bucketeer/values.yaml:2529]: please remove `bigtableInstance`

File: manifests/bucketeer/values.yaml [hunk 2]
```diff
@@ -2513,3 +2513,133 @@ web-gateway:
...
+    pullPolicy: IfNotPresent
+  nameOverride: "event-persister-dwh"
+  fullnameOverride: "event-persister-goal-events-dwh"
+  namespace: default
+  env:
+    project:
+    logLevel: info
+    port: 9090
+    metricsPort: 9002
+    bigtableInstance:
```
Comment (masaaania) [inline manifests/bucketeer/values.yaml:2594]: please remove `bigtableInstance`

File: manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+              port: service
+              scheme: HTTPS
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.health.initialDelaySeconds }}
+            httpGet:
+              path: /health
+              port: service
+              scheme: HTTPS
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
```
Comment (cre8ivejp) [inline manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml:114]: Please use `nindent` instead.
Same for the other places.

E.g.
https://github.com/bucketeer-io/bucketeer/pull/154/files#diff-9dda85b788b55f6d80b4593f70270c815fd495cdb8fc2e82192cc955da9c9959R25

File: manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+          image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: ["server"]
+          env:
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PROJECT
+              value: "{{ .Values.env.project }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_EXPERIMENT_SERVICE
+              value: "{{ .Values.env.experimentService }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_BIGTABLE_INSTANCE
+              value: "{{ .Values.env.bigtableInstance }}"
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml:52]: please remove `bigtableInstance`

File: manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+      volumes:
+        - name: envoy-config
+          configMap:
+            name: {{ template "event-persister-evaluation-events-dwh.fullname" . }}-envoy-config
+        - name: service-cert-secret
+          secret:
+            secretName: {{ template "service-cert-secret" . }}
+        - name: service-token-secret
+          secret:
+            secretName: {{ template "service-token-secret" . }}
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml:40]: please insert `serviceAccountName: {{ template "event-persister-evaluation-events-dwh.name" . }}`

**Review by cre8ivejp — CHANGES_REQUESTED** [review]

File: manifests/bucketeer/charts/event-persister-evaluation-events-dwh/values.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+fullnameOverride: "event-persister-evaluation-events-dwh"
+
+namespace:
+
+env:
+  project:
+  logLevel: info
+  port: 9090
+  metricsPort: 9002
+  bigtableInstance:
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-evaluation-events-dwh/values.yaml:15]: please remove `bigtableInstance`

File: manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+          image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: ["server"]
+          env:
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PROJECT
+              value: "{{ .Values.env.project }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_EXPERIMENT_SERVICE
+              value: "{{ .Values.env.experimentService }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_BIGTABLE_INSTANCE
+              value: "{{ .Values.env.bigtableInstance }}"
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/deployment.yaml:52]: please remove `bigtableInstance`

File: manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/deployment.yaml [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+      volumes:
+        - name: envoy-config
+          configMap:
+            name: {{ template "event-persister-goal-events-dwh.fullname" . }}-envoy-config
+        - name: service-cert-secret
+          secret:
+            secretName: {{ template "service-cert-secret" . }}
+        - name: service-token-secret
+          secret:
+            secretName: {{ template "service-token-secret" . }}
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/deployment.yaml:40]: please insert `serviceAccountName: {{ template "event-persister-goal-events-dwh.name" . }}`

File: manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/service-cert-secret.yaml [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+  labels:
+    app: {{ template "event-persister-goal-events-dwh.fullname" . }}
+    chart: {{ template "event-persister-goal-events-dwh.chart" . }}
+    release: {{ template "event-persister-goal-events-dwh.fullname" . }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  tls.crt: {{ required "Service TLS certificate is required" .Values.tls.service.cert | b64enc | quote }}
+  tls.key: {{ required "Service TLS key is required" .Values.tls.service.key | b64enc | quote }}
+{{- end }}
```
Comment (masaaania) [inline manifests/bucketeer/charts/event-persister-goal-events-dwh/templates/service-cert-secret.yaml:16]: please add an empty line at the end.

**Comment by masaaania** [comment]
We also need to create `charts/event-persister-{evaluation|goal}-events-dwh/templates/service-account.yaml` files.

**Comment by ono-max** [comment]
@cre8ivejp @masaaania 

Updated 👍

File: manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml (hunk 5 above)
Comment (masaaania) [reply manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml:39]: Sorry for the small detail, can you insert `serviceAccountName` between `volumes` and `containers` like other files for consistency?

eg. https://github.com/bucketeer-io/bucketeer/blob/1d86fe83f19333b0c106ecd967fb4f7cf280f291/manifests/bucketeer/charts/event-persister-evaluation-events-kafka/templates/deployment.yaml#L31-L42

**Comment by ono-max** [comment]
@cre8ivejp @masaaania

Updated 👍

**Review by cre8ivejp — DISMISSED** [review]
Looks good!
Thanks!

**Review by masaaania — DISMISSED** [review]
LGTM! Thank you!

**Comment by ono-max** [comment]
@cre8ivejp 

I resolved conflicts. Could you approve this PR again? Thank you!

File: tools/build/show-image-name.sh [hunk 10]
```diff
@@ -6,6 +6,8 @@ case $APP in
     "autoops") echo "auto-ops" ;;
     "eventcounter") echo "event-counter" ;;
     "eventpersister") echo "event-persister" ;;
+    "eventpersister-dwh") echo "event-persister-dwh" ;;
+    "goalbatch") echo "goal-batch" ;;
```
Comment (cre8ivejp) [inline tools/build/show-image-name.sh:10]: Please remove this line.

**Comment by ono-max** [comment]
@cre8ivejp 

Thank you, updated 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you!!

nit: `chore: add manifest files for event persister dwh`

**Pull request merged**